Makefile

```diff
@@ -50,18 +69,29 @@ update:
 update-all:
 	@for dir in models/*/ ; do \
 		cp model_templates/predict.py $$dir ; \
-		cp model_templates/cog.yaml $$dir ; \
+		#cp model_templates/cog.yaml $$dir ; \
```

Comment:
should this be commented?

Reply:
if we can't remove trl for 70b training then the cog.yaml have to stay different for now, so it doesn't make sense to update all of them at once. anyway this target isn't necessary after all